paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -14,8 +14,8 @@ import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.toArgb
 import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.compose.ui.res.dimensionResource
```

Comment:
GooglePayButton uses primary button (see stripe_google_pay_button.xml), so I don't think you can hard code the style here.

Reply:
For now since their heights are not dynamic they can both share the same resource value. They both should be the same height right?

Reply:
I think this will have the effect of changing the google pay button only while in the processing state.

Reply:
Good point. It does look off when I slowed animations down. I reverted all of the height related changes I made and now it is the same as it was before I started this work flow. We dropped the dynamic height from the api so this is totally not needed now. 

PR that originally added this height stuff: https://github.com/stripe/stripe-android/pull/4869/files